Gemfile

```diff
@@ -10,6 +10,7 @@ platform :mri do
   gem 'webrick'
   gem "typhoeus", "~> 1.1.0"
   gem "patron", "0.6.3"
+  gem "ethon", "0.15.0" # TODO: Remove this pinned dependency once spec issue with Typhoeus is resolved.
```

Comment:
Could the URL to the issue be in this code comment? Thanks!

Reply:
It's only been reported on Typhoeus (is https://github.com/typhoeus/typhoeus/issues/705 the issue you're referencing?), looking into`ethon` now - seems the root of the issue has to do with [this change](https://github.com/typhoeus/ethon/commit/da1a541f5f875fcffa6c3e73cf266595e9de8ddf).